truenas/webui
# NAS-118602 / 22.12 / Add ability for users to fix clock

**Testing**

1. On your VM box shift the clock so it differs from your host machine clock
2. Check System Info widget: warning should be displayed
3. Click "Synchronize time" button and check that the time was updated
